File: Programs/OOP Assignments/Chapter 3/student.cpp
#include <iostream>
#include <string.h>
using namespace std;

class student

{

	int roll_no;

	char name[20];

	float percentage;

public:
	student(int rno, char *n, float per)

	{

		roll_no = rno;

		strcpy(name, n);

		percentage = per;
	}

	void display()

	{

		cout << "\n Roll No:" << roll_no;

		cout << "\n Name:" << name;

		cout << "\n Percentage:" << percentage;
	}
};

int main()

{

	student s1(112, " Chitrakshi", 89), s2(114, "Anjali", 98);

	s1.display();

	s2.display();

	return 0;
}

File: SRC/JPR/student.java
class student {
	int id;
	String name;

	student(int i, String n) {
		id = i;
		name = n;
	}

	student(student s)// copy constructor
	{
		id = s.id;
		name = s.name;
	}

	void display() {
		System.out.printf(id + "" + name);
	}

	public static void main(String args[]) {
		student s1 = new student(123, " ABC ");
		s1.display();
		student s2 = new student(s1);
		s2.display();
	}
}